spel/minimal-linux.json

```
@@ -452,6 +452,7 @@
                 "SPEL_AMIGENMANFST={{ user `spel_amigen8manfst` }}",
                 "SPEL_AMIGENREPOS={{ user `spel_amigen8reponames` }}",
                 "SPEL_AMIGENREPOSRC={{ user `spel_amigen8reposource` }}",
+                "SPEL_AMIGEN8_ROOTNM={{ user `spel_amigen8_rootnm` }}",
```

We don't actually have a variable named `spel_amigen8_rootnm` that I see? Am I just missing it?

---

from amigen8.sh 
`AMIGENROOTNM="${SPEL_AMIGEN8_ROOTNM:-UNDEF}"`

---

It is for setting or not setting up LVM
`   # Set LVM2 or bare disk-formatting
   if [[ ${AMIGENVGNAME} != "UNDEF" ]]
   then
      DISKSETUPCMD+="-v ${AMIGENVGNAME} "
   elif [[ ${AMIGENROOTNM} != "UNDEF" ]]
   then
      DISKSETUPCMD+="-r ${AMIGENROOTNM} "
   fi`

---

That's in amigen8.sh though, and is setup so when not passed the value is `UNDEF`. The change here is using

```
user `spel_amigen8_rootnm`
```

which is how packer references packer variables. But I don't see `spel_amigen8_rootnm` in the map of packer `variables` in the json template.

And actually, looking at the other `variables` conventions, the variable name should probably be `spel_amigen8rootnm`...

```suggestion
                "SPEL_AMIGEN8_ROOTNM={{ user `spel_amigen8rootnm` }}",
```

---

I see what you meant. My fault I missed updating there too. I have made the changes including the more appropriate variable name.